.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
We don't run prettier directly as we use a plugin in eslint for that, so all configurations are inherited from the external lib of eslint, so we don't need to configure it.

I am curious why you had an issue here.

Reply:
Ah, I see! My normal setup is to only use a VSCode Prettier extension and _format on save_, ESLint I have disabled.

Turning on the ESLint extension, _and_ either specifying it as the default formatter on _workspace_ level, or disabling the default formatter on _user_ level solves the issue


I'll remove the _.prettierrc_ file